lukefi/metsi/domain/pre_ops.py

```diff
@@ -71,6 +71,7 @@ def generate_reference_trees(stands: list[ForestStand], **operation_params) -> l
     debug_tree_rows = []
     stratum_association_diameter_threshold = operation_params.get('stratum_association_diameter_threshold', 2.5)
     for i, stand in enumerate(stands):
+        stand.area_weight = stand.area * stand.area_weight_factors[1]
```

Comment:
This actually does not work correctly in the case of auxiliary stands because we set the stand.area to zero in ForestBuilder. stand.area could be replaced with stand.area_weight, but this involves a risk that stand.area_weight does not have the expected value (the original area).

Reply:
Moved the logic into a new operation level function called `scale_area_weight`. Added into `control.yaml`.